src/traces/sunburst/attributes.js

```diff
@@ -203,6 +203,18 @@ module.exports = {
             'this option won\'t have any effect and `insidetextfont` would be used.'
         ].join(' ')
     }),
+    rotation: {
+        valType: 'number',
```

Comment:
I'm not sure why pie didn't use this, but there's also a [`valType: 'angle'`](https://github.com/plotly/plotly.js/blob/b521323ab99fd0d6679e9756a19223e053d6b737/src/lib/coerce.js#L176-L187) that has the nice property of regularizing any number to [-180, 180], so for example you can provide 270 and we'll convert it to -90, giving the desired result.

The one complication of this valType is it currently always allows the special value `'auto'` that we'd have to explicitly treat as 0 - for example the `if` statement I suggested below could become `if(trace.rotation && trace.rotation !== 'auto')`